docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -392,3 +392,16 @@ Payload:
     ...
 }
 ```
+
+## Skip organization role sync
```

Comment:
@eleijonmarck , the formatting of this text (it's in blue?) makes me think something is incorrect futher up in the doc. Please check Line 220. I think the quotes need to be a > symbol?